modules/kuberay-cluster/values.yaml

```diff
@@ -36,7 +36,7 @@ head:
   # If enableInTreeAutoscaling is true, the autoscaler sidecar will be added to the Ray head pod.
   # Ray autoscaler integration is supported only for Ray versions >= 1.11.0
   # Ray autoscaler integration is Beta with KubeRay >= 0.3.0 and Ray >= 2.0.0.
-  # enableInTreeAutoscaling: true
+  enableInTreeAutoscaling: true
```

Comment:
thanks for re-enabling it! Can we add a small section to the readme that details how to use the autoscaler? for eg, for a larger dataset, maybe a user would need more than the max allowed workers (5) and they should know how to change it.

Reply:
Before this PR, the replicas of the worker was set to 1 so the current change can probably cover our previous usage assumption. I'm a little hesitating to add more description around how to change these configuration because for the configurations in this file, we don't expose them to users (i.e. via workloads.tfvars) and we don't have a README of how/when to changing them, which gave me impression that we don't want or at least don't expect users to change them, since these configurations work with the example we provide. 
For you particular example, since right now we don't provide guidance on how to change the dataset, I think adding guidance on changing the configurations may not necessary.
But again, I don't have strong opinion. :)

Reply:
Or, we want to expose a variable in "application/rag/workloads.tfvars" and "application/ray/workloads.tfvars" to give users control to enable/disable the ray autoscaler? I feel that's a little over engineering for the purpose of the example application. And I didn't see much concern to just use ray autoscaler as the solution.